crrExistingObjects.js

```diff
@@ -223,13 +240,45 @@ function _listObjectVersions(bucket, VersionIdMarker, KeyMarker, cb) {
         Prefix: TARGET_PREFIX,
         VersionIdMarker,
         KeyMarker,
-    }, cb);
+    }, (err, data) => {
+        if(err) {
+            // retry only on Network errors and if the process is set to retry
+            // on errors.
+            if (err.code && err.code === 'NetworkingError' && RETRY_ON_ERROR) {
+                const errMsg = `Error listing object versions for bucket: ` +
+                                `${bucket}, retrying request..`;
```

Comment:
No need for double or single punctuation or caps.

Please don't make variable error messages, instead embed the bucket name in the log message attributes (`{ error: err, bucketName: bucket }`), to make it searchable by attribute instead of full-text.

For more information on logging guidelines, please see the scality/guidelines github repo.

Reply:
@RachedBenMustapha I pushed this commit to check linter errors :), you are quick!

Reply:
Corrected.